src/reducers/userEventsReducer.js

```diff
@@ -0,0 +1,14 @@
+export const userEventsReducer = (state = [], action) => {
+  switch(action.type) {
+    case 'LOAD_EVENTS': 
+     return action.events
+    case 'ADD_EVENT':
+      return [...state, action.event]
+    case 'REMOVE_EVENT':
+      return state.filter(event => event.event_id !== action.event_id)
+    case 'EDIT_EVENT':
+      return state.map(event => {
+        return event.event_id === action.event_id ? { ...action.event } : event
+      })
+  }
```

Comment:
Nicely done on this reducer!! Good use of switch cases/ternarys/implicit returns to write clean, readable code